### ISSUE TITLE
Get checkbox with accordion menu

### ISSUE THREAD

**Issue by AnotherAnkor** [nolimits4web/Framework7#585]
Hi everyone!
I need something like accordion menu with checkboxes. It means I can set checkbox and/or open accordion menu and check/uncheck some elements inside.
Anybody can help me?

**Comment by nolimits4web** [comment]
You need to use Accordion List View (http://www.idangero.us/framework7/docs/accordion.html#accordion-list-view) with Checkboxes and Radios (http://www.idangero.us/framework7/docs/checkboxes-radios.html#.VdOBn1Ptmko)

**Comment by lionel-bijaoui** [comment]
I've done just that and there is a problem with the CSS selector. Here is a checkbox+accordion that contain a checkbox.

    <label class="label-checkbox item-content">
        <input type="checkbox">
        <div class="item-media">
            <i class="icon icon-form-checkbox"></i>
        </div>
        <div class="item-inner">
            <div class="accordion-item">
                <a href="#" class="item-content item-link">
                    <div class="item-inner">
                        <div class="item-title">Item 1</div>
                    </div>
                </a>
                <div class="accordion-item-content">
                    <div class="list-block">
                        <ul>
                            <li>
                                <label class="label-checkbox item-content">
                                    <input type="checkbox">
                                    <div class="item-media">
                                        <i class="icon icon-form-checkbox"></i>
                                    </div>
                                    <div class="item-inner">
                                        <div class="item-title">Sub item 1</div>
                                    </div>
                                </label>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>
        </div>
    </label>

The problem is that if I check the outer checkbox, the inner checkbox appear checked. The real value of the hidden checkbox html input is correct, it is only a problem with selector.

For ```.icon-form-checkbox``` the selector ```label.label-checkbox input[type=checkbox]:checked+.item-media i.icon-form-checkbox``` is used in a good way and don't cause problem.
But the other selector ```label.label-checkbox input[type=checkbox]:checked~.item-inner i.icon-form-checkbox``` is too broad and influence the inner selector as well.
A simple fix I found is to add ```>``` like so ```label.label-checkbox input[type=checkbox]:checked~.item-inner>i.icon-form-checkbox```.

I don't know how to do pull request, but it would be nice to fix this for everyone. Except, of course, if I made a mistake and then I apologize in advance.

**Comment by nolimits4web** [comment]
This code can't be valid by idea as you have `label>label>input` You probably need to use the basic structure from docs:
```
<li class="accordion-item">
  <label class="label-checkbox item-content accordion-item-toggle">
    ...
  </label>
  <div class="accordion-item-content">
    ..nested UL here
  </div>
</li>
```

**Comment by lionel-bijaoui** [comment]
Sorry for the confusion, I don't want to use checkbox as open/close indicator.   
I want an independent checkbox and accordion. I can check/uncheck or open/close the accordion, but I want them visually in the same space (without rewriting a lot of CSS).
I understand that this is a bit unconventional, but it work. I will take into account that `label>label>input` is semantically incorrect (very good point) but since this is an hybrid app, it should not be a big problem for now.
I will have to rewrite a lot of CSS in the end, but for now I want a working prototype.
Thank you for your help and comment.

**Comment by nolimits4web** [comment]
The problem in `label>label>input` is not the semantic. It is about how the label works. If you click parent label it will check/uncheck all nested inputs. Anyway, is this problem is still actual?

**Comment by lionel-bijaoui** [comment]
The only problem I was facing was that the "checked" attribute was not changed in the sub checkboxes, but their aspect was, and I needed the attribute to changed. I implemented the good behavior with js, and then, the css was not a problem. I also added a change when some sub checkbox are selected.
![untitled-1](https://cloud.githubusercontent.com/assets/3472596/10720551/e7636dba-7b98-11e5-8538-f03d1b5a8e1c.png)
I'm clearly working outside of the standard use, so it is not a problem with the framework really. It work as indented and I shouldn't have bothered you with that in the first place, thank you for your help and time.
